lnd_test.go

```diff
@@ -1,5 +1,8 @@
 // +build rpctest
 
+// TODO(joostjager): Assert on structured errors instead of
```

Comment:
As the router rpc is still in experimental state and its interface may change, we could also wait until we actually deprecate the main `SendToRoute` rpc.

Changing it now also moves coverage away from the main rpc and it may be safer to not yet do that.